src/App.tsx

```diff
@@ -1,48 +1,44 @@
+import { NavLink, Routes, Route } from 'react-router-dom';
+import { HomePage } from './components/HomePage/HomePage';
+import { TabsPage } from './components/TabsPage/TabsPage';
 import 'bulma/css/bulma.css';
-import '@fortawesome/fontawesome-free/css/all.css';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
 
 const App = () => (
-  <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav className="navbar is-fixed-top has-background-light" data-cy="nav">
-      <div className="navbar-menu">
-        <div className="navbar-start">
-          <a href="/" className="navbar-item isActive">Home</a>
-          <a href="/tabs" className="navbar-item isActive">Tabs</a>
-        </div>
-      </div>
-    </nav>
-
-    <div className="section">
-      <h1 className="title">Home page</h1>
-      <h1 className="title">Tabs page</h1>
-      <h1 className="title">Page not found</h1>
+  <div className="App container">
+    <nav className="navbar">
+      <NavLink
+        className={({ isActive }) => (
+          `navbar-item is-tab ${isActive ? ' is-active' : ''}`
+        )}
+        to="/"
+      >
+        Home Page
+      </NavLink>
 
-      <div className="tabs is-boxed">
-        <ul>
-          <li data-cy="tab" className="is-active">
-            <a href="#/">Tab 1</a>
-          </li>
-          <li data-cy="tab">
-            <a href="#/">Tab 2</a>
-          </li>
-          <li data-cy="tab">
-            <a href="#/">Tab 3</a>
-          </li>
-        </ul>
-      </div>
+      <NavLink
+        className={({ isActive }) => (
+          `navbar-item is-tab ${isActive ? ' is-active' : ''}`
+        )}
+        to="/tabs"
+      >
+        Tabs
+      </NavLink>
+    </nav>
 
-      <div className="block" data-cy="tab-content">
-        Please select a tab
-      </div>
-    </div>
-  </>
+    <main>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="tabs/:tabId" element={<TabsPage tabs={tabs} />} />
+        <Route path="/tabs" element={<TabsPage tabs={tabs} />} />
+      </Routes>
```

Comment:
use nested routes
for more info, learn this article – https://reactrouter.com/docs/en/v6/getting-started/concepts#index-routes

Reply:
also – don't forget about it
![Screen Shot 2022-08-05 at 09 23 20](https://user-images.githubusercontent.com/75496546/183014648-bd2d74a0-1eac-41cd-8ef1-811deaad1d26.png)

Reply:
Fixed